Gruntfile.js

```diff
@@ -13,6 +15,9 @@ module.exports = function(grunt) {
 
   grunt.initConfig({
     pkg: pkg,
+    webpack: {
+      options: webpackConfig
+    },
     jshint: {
       options: {
         globals: {
```

Comment:
There's lots of stuff below that we won't need anymore. E.g. the 'concat/amd/commonjs' targets.